gnomad_qc/v5/resources/sample_qc.py

```diff
@@ -170,3 +170,21 @@ def get_cuking_output_path(
     """
     # cuKING outputs can be easily regenerated, so use a temp location.
     return f"{qc_temp_prefix(version=version, environment=environment)}cuking_output{'_test' if test else ''}.parquet"
+
+
+def relatedness(test: bool = False) -> VersionedTableResource:
```

Comment:
it looks like `relatedness(method="cuking").ht()` returns something different from `relatedness().ht()` for [v4](https://github.com/broadinstitute/gnomad_qc/blob/8b8c766539c682f6a1596d019e44951440119a1a/gnomad_qc/v4/resources/sample_qc.py#L476) -- maybe we should keep `method` or rename that argument to be more informative

Reply:
Yes I'll revert to that or think of a more informative